cargo-dist/src/lib.rs

```diff
@@ -173,11 +173,14 @@ fn run_build_step(
     Ok(())
 }
 
-const AXOUPDATER_ASSET_ROOT: &str =
-    "https://github.com/axodotdev/axoupdater/releases/latest/download";
+const AXOUPDATER_ASSET_ROOT: &str = "https://github.com/axodotdev/axoupdater/releases/";
 const AXOUPDATER_MINIMUM_VERSION: &str = "0.7.0";
 const AXOUPDATER_GIT_URL: &str = "https://github.com/axodotdev/axoupdater.git";
 
+fn axoupdater_asset_root() -> String {
+    format!("{AXOUPDATER_ASSET_ROOT}/v{}/download", axoupdater::VERSION)
```

Comment:
does this prevent auto-updates?

Reply:
No, this is where we'll fetch standalone axoupdater builds from. It just means a given release of dist will always install a specific version of the standalone updater when packaging software, instead of a floating version. dist's own autoupdates will work.

Reply:
ah sorry i just realized how vague what i wrote was... i meant to ask if this means that a single version of dist will only package a single version of axoupdater and to use a new version of axo updater we'd need to make a new dist release

Reply:
Ah, got it. Yes, I felt that this was probably a good thing at this point - protection against breaking changes without a corresponding dist release, now that axoupdater is out of its initial experimental phase.